src/common/CircleGameObject.cpp

```diff
@@ -1,18 +1,38 @@
 #include "CircleGameObject.hpp"
 
 
-CircleGameObject::CircleGameObject(unsigned int id, sf::Vector2f position, float size)
-    :GameObject(id), mCircle(std::shared_ptr<Circle>(new Circle(size, position))) {}
+CircleGameObject::CircleGameObject(
+    unsigned int id, sf::Vector2f position, float radius, std::shared_ptr<CircleRigidBody> crb):
```

Comment:
Maybe rather than passing in a CircleRigidBody, we should pass a reference of the PhysicsContoller and have this class initialize it's own CircleRigidBody via createCRB()

Reply:
Made this change 93b3e79